bridgeservice/bridge.go

```diff
@@ -147,8 +143,8 @@ func (b *BridgeService) registerRoutes() {
 		bridgeGroup.GET("/l1-info-tree-index", b.L1InfoTreeIndexForBridgeHandler)
 		bridgeGroup.GET("/injected-l1-info-leaf", b.InjectedL1InfoLeafHandler)
 		bridgeGroup.GET("/claim-proof", b.ClaimProofHandler)
-		bridgeGroup.POST("/sponsor-claim", b.SponsorClaimHandler)
-		bridgeGroup.GET("/sponsored-claim-status", b.GetSponsoredClaimStatusHandler)
+		// bridgeGroup.POST("/sponsor-claim", b.SponsorClaimHandler)
+		// bridgeGroup.GET("/sponsored-claim-status", b.GetSponsoredClaimStatusHandler)
```

Comment:
Probably it should be enough just updating the `default.go` and disabling the claim sponsor, we don't need to remove the implemenation. WDYT?